proto/eth/service/key_management.proto

```diff
@@ -130,7 +130,8 @@ message DeletedKeystoreStatus {
   enum Status {
     DELETED = 0;
     NOT_FOUND = 1;
-    ERROR = 2;
```

Comment:
Changing enum value IDs is a really bad idea. 
If the server has the old proto and client has the new one, you're going to get unexpected behavior.